tests/message/test_messages.py

```diff
@@ -345,3 +345,10 @@ def test_message_remove_task_container(self):
             )
         for i in range(test_cases):
             self.assertEqual(msg_l.remove_tasks[i].task_id, task_ids[i])
+
+    def test_message_task_to_compute_with_luxrender_script_package(self):
```

Comment:
what about `blender_script_package`? It seems that there is no test for that.

Reply:
It is tested in every other messages because `blender_script_package` is a default package for `ComputeTaskDef`. But yea, why not. Never enough tests :) Done

Reply:
IMO this should be moved to `tests/message/te_tasks.py`